### ISSUE TITLE
Browser Full Version

### ISSUE THREAD

**Issue by goldenratio** [lancedikson/bowser#172]
Hi,
`bowser.version` returns the major version. Do we have a feature to return full version.

Example useragent,
```
Mozilla/5.0 (Windows NT 6.3; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/55.0.2883.87 Safari/537.36
```

bowser.version returns 55.0, but is it possible to get 55.0.2883.87

**Comment by lancedikson** [comment]
No, now it's not. But, let's make it possible. Do you wish to make a PR for it? :)

**Comment by leesc81** [comment]
i need this feature too

**Comment by lancedikson** [comment]
Released in 2.0.0-alpha.1
